jena-fuseki2/jena-fuseki-core/pom.xml

```diff
@@ -96,11 +96,17 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.jena</groupId>
```

Comment:
This gets pulled in transitively via `jena-arq`. You shouldn't need to add it explicitly. Try removing it and nothing should change.